addons/mail/static/src/new/chat/chat_window_icon.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+    <t t-name="mail.chat_window_icon" owl="1">
+        <div class="o-mail-chatwindow-icon d-flex justify-content-center flex-shrink-0 ms-3 me-1 my-0">
+            <t t-if="thread.type === 'channel'">
+                <div t-if="thread.authorizedGroupFullName" class="o_ThreadIconView_groupRestrictedChannel fa fa-fw fa-hashtag" t-att-title="thread.accessRestrictedToGroupText"/>
+                <div t-if="!thread.authorizedGroupFullName" class="o_ThreadIconView_publicChannel fa fa-fw fa-globe" title="Public Channel"/>
+            </t>
+            <t t-elif="thread.type === 'chat' and chatPartner">
+                <Typing t-if="isTyping" channel_id="thread.id" size="'small'" displayText="false"/>
```

Comment:
Instead of `displayText`, I think it found be best a `mode` as prop of sort sort.
Not a big deal deal, it works which is ok

Reply:
Well if this component had 3 modes I would be nice, but this is just a option to display something or not. If I use mode, I don't understand what option other than 'text'

Reply:
I was thinking of text vs icon-only or something like that?